tests/Silverback.Core.Tests/Util/TypesCacheTests.TestData.cs

```diff
@@ -0,0 +1,100 @@
+                 new AssemblyQualifiedGenericTypeResult(
+                     "Silverback.Tests.Core.Util.TypesCacheTests+GenericTypeTest`2[[System.Int64, System.Private.CoreLib],[Silverback.Tests.Core.Util.TypesCacheTests, Silverback.Core.Tests]], Silverback.Core.Tests",
+                     typeof(GenericTypeTest<long, TypesCacheTests>)),
+            };
+
+            // Shortened qualified name with types [shortened qualified name],[full qualified name]
+            yield return new object[]
+            {
+                 "Silverback.Tests.Core.Util.TypesCacheTests+GenericTypeTest`2[[System.Int64, System.Private.CoreLib],[Silverback.Tests.Core.Util.TypesCacheTests, Silverback.Core.Tests, Version=1.2.3.4, Culture=neutral, PublicKeyToken=null]], Silverback.Core.Tests",
+                 new AssemblyQualifiedGenericTypeResult(
+                     "Silverback.Tests.Core.Util.TypesCacheTests+GenericTypeTest`2[[System.Int64, System.Private.CoreLib],[Silverback.Tests.Core.Util.TypesCacheTests, Silverback.Core.Tests]], Silverback.Core.Tests",
+                     typeof(GenericTypeTest<long, TypesCacheTests>)),
+            };
+
+            // Full qualified name with full qualified types.
+            yield return new object[]
+            {
+                 "Silverback.Tests.Core.Util.TypesCacheTests+GenericTypeTest`2[[System.Int64, System.Private.CoreLib, Version=1.2.3.4, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[Silverback.Tests.Core.Util.TypesCacheTests, Silverback.Core.Tests, Version=1.2.3.4, Culture=neutral, PublicKeyToken=null]], Silverback.Core.Tests, Version=1.2.3.4, Culture=neutral, PublicKeyToken=null",
+                 new AssemblyQualifiedGenericTypeResult(
+                     "Silverback.Tests.Core.Util.TypesCacheTests+GenericTypeTest`2[[System.Int64, System.Private.CoreLib],[Silverback.Tests.Core.Util.TypesCacheTests, Silverback.Core.Tests]], Silverback.Core.Tests",
+                     typeof(GenericTypeTest<long, TypesCacheTests>)),
+            };
+
+            // Full qualified name - no culture, with type [full qualified name - no version]
+            yield return new object[]
+            {
+                 "System.Collections.Generic.List`1[[System.Char, System.Private.CoreLib, Culture=neutral]], System.Private.CoreLib, Version=5.0.0.0, PublicKeyToken=7cec85d7bea7798e",
+                 new AssemblyQualifiedGenericTypeResult(
+                     "System.Collections.Generic.List`1[[System.Char, System.Private.CoreLib]], System.Private.CoreLib",
+                     typeof(List<char>)),
+            };
+        }
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1034:Nested types should not be visible", Justification = "Record belongs to unit test data.")]
+        public record AssemblyQualifiedGenericTypeResult(string CleanAssemblyQualifiedName, Type MatchingType)
+        {
+        }
+
+        /// <summary>
```

Comment:
Unecessary summary, you can remove it.

Reply:
It's still there though. 😆

Reply:
Sorry, I'm dumb, confused it with the other summary 🤣